### ISSUE TITLE
About the code for the add&norm

### ISSUE THREAD

**Issue by Daniel-Jiang358** [mattsherar/Temporal_Fusion_Transform#13]
The code is very clear to read, but I find it confusing that in the class TFT,

 self.post_lstm_norm = TimeDistributed(nn.BatchNorm1d(self.hidden_size))

So just batch norm is okay there? I don't see the parameters of the batch_first there. Thanks for you kind reply.
